velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -378,3 +379,23 @@ TEST_F(PrestoSerializerTest, lazy) {
       std::make_unique<SimpleVectorLoader>([&](auto) { return rowVector; }));
   testRoundTrip(lazyVector);
 }
+
+TEST_F(PrestoSerializerTest, ioBufRoundTrip) {
+  serializer::presto::PrestoVectorSerde::registerVectorSerde();
+
+  VectorFuzzer::Options opts;
+  opts.timestampPrecision =
+      VectorFuzzer::Options::TimestampPrecision::kMilliSeconds;
+  VectorFuzzer fuzzer(opts, pool_.get());
+
+  const size_t numRounds = 20;
+
+  for (size_t i = 0; i < numRounds; ++i) {
+    auto rowType = fuzzer.randRowType();
+    auto inputRowVector = fuzzer.fuzzRow(rowType);
```

Comment:
fuzzInputRow; I don't think we support serializing null top-level rows

Reply:
Good call. I wondered why this hasn't failed and turned out that we don't generate nulls in fuzzer by default. I fixed and asked fuzzer to generate some nulls, just in case.